bin/ch/Helpers.cpp

```diff
@@ -42,81 +41,60 @@ HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCSTR& contents, bool* isU
         IfFailGo(E_FAIL);
     }
 
-    //
-    // Determine the file length, in bytes.
-    //
-    fseek(file, 0, SEEK_END);
-    lengthBytes = ftell(file);
-    fseek(file, 0, SEEK_SET);
-    pRawBytes = (LPBYTE)malloc(lengthBytes + sizeof(WCHAR));
-    if (nullptr == pRawBytes)
+    if (file != nullptr)
```

Comment:
nit: Is this check necessary? If yes, maybe can reorder the code a bit to make diff tool happy so it is easier to see what's changed.